### PR TITLE
EREGCSC-1562 - Implement Snyk

### PR DESCRIPTION
**Pull request by KaushikSamtek**
Resolves # [EREGCSC-1562](https://jiraent.cms.gov/browse/EREGCSC-1562)

**Description** 
This PR is for Snyk implementation. It contains the workflow to run the scan and then calls the service that creates the Jira tickets.

**This pull request changes...**

This pull request changes none of the existing files. Only one new workflow file was added.

**Steps to manually verify this change**

When a pull request is made to main, there will be a Snyk run in the actions tab.

### CODE REVIEW

**Comment by KaushikSamtek** [comment]
Sorry, I just had to remove push from the title for snyk. No major changes were made.

File: .github/workflows/snyk-test.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+jobs:
+  snyk_run:
+    name: Snyk Run (for PR)
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
+    
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
```
Comment (thwalker6) [inline .github/workflows/snyk-test.yml:20]: We currently use checkout@v3 on our application since that is the latest version.  Please update.

**Review by thwalker6 — CHANGES_REQUESTED** [review]
Please update the github actions for checkout to v3.

**Comment by KaushikSamtek** [comment]
I have updated the version to v3. I also saw this branch was a commit behind main so I updated that also and not it is ready to go. Let me know if there is anything else

**Review by thwalker6 — APPROVED** [review]
lgtm

**Pull request merged**